tensorflow_addons/optimizers/__init__.py

```diff
@@ -14,6 +14,7 @@
 # ==============================================================================
 """Additional optimizers that conform to Keras API."""
 
+from tensorflow_addons.optimizers.constants import BASE_OPTIMIZER_CLASS
```

Comment:
this breaks alphabetic order

Reply:
This order matters, if this line does not go before other optimizers it creates a cyclic importing.